### DISCUSSION TITLE
Day in test_x

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#188] [BaiduKDDCup2022] [answered]
If day number in test_x with the same reference than train data. I mean, in test_x the day will be >245 or will it start again in 1. Thank you.

**Comment by ramber1836** [comment, marked as answer]
The Day column of each test_x file starts from 1, which means that the day in test_x is relative to the beginning of each prediction period. Please take the toy data as an example (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_test_toy).
